structure/deque/deque.go

```diff
@@ -0,0 +1,91 @@
+/*
+	Package deque implements a Double Ended Queue data structure.
```

Comment:
Package comment will probably get converted to code block, remove the unnecessary indentation.